WooCommerce/Classes/ViewRelated/Products/Edit Product/Inventory Settings/ProductInventorySettingsViewController.swift

```diff
@@ -390,6 +425,24 @@ private extension ProductInventorySettingsViewController {
     func rowAtIndexPath(_ indexPath: IndexPath) -> Row {
         return sections[indexPath.section].rows[indexPath.row]
     }
+
+    func getIndexPathForRow(_ row: Row) -> IndexPath? {
```

Comment:
The readability is a little harder with single-letter variables. 

I suggest using `sectionIndex` instead of `i` and `rowIndex` instead of `j`. 

Or if you prefer to stick with single-letter variables, use the same rules that algebra does. (For example, in algebra we say "given function `f`..." meaning the function name is f). We are finding section `s` in `sections` and row `r` in `rows`. That way it shows the relationship between `r` and `rows` and `s` with `sections` when it's read:

```
return IndexPath(rows: r, section: s)
```